js/offcanvas.js

```diff
@@ -36,14 +36,14 @@
     if (this.options.autohide && !this.options.modal) {
       var eventName = (navigator.userAgent.match(/(iPad|iPhone)/i) === null) ? 'click' : 'touchstart'
       $(document).on('click touchstart', $.proxy(this.autohide, this))
-    }
-
-    if (this.options.toggle) this.toggle()
+    }   
 
-    if (this.options.disablescrolling) {
+    if (typeof(this.options.disablescrolling) === "boolean") {
```

Comment:
You're using jQuery .data() which converts "false" to false (boolean) automatically.

And if you pass "disablescrolling": false in JS, everything works fine, I am not sure what you meant here ?

I tested this PR and this works fine for me.